test/ParseExceptionTest.cpp

```diff
@@ -40,4 +40,7 @@ TEST(ParseException, MetadataGeneration) {
   // from the Lexer. There is no ExceptionMetadata available for Lexer Errors.
   expectParseExceptionWithMetadata("SELECT * WHERE { % }",
                                    {{"where  { % }", 9, 9, 1, 9}});
+  expectParseExceptionWithMetadata(
+      "SELECT * WHERE {\n ?a ?b ?c . \n f:d ?d ?e\n}",
+      {{"where  {\n ?a ?b ?c . \n f:d ?d ?e\n}", 23, 25, 3, 1}});
```

Comment:
I expect that the error here is, that the prefix `f` is undefined. After the newline there is a space. Is the `positionInLine` 0-based, or does the whitespace not count, because the of the `skip` action in the lexer?

Reply:
`positionInLine` is 0-based. The positions dont skip anything relative to `query_`.

Reply:
Great, thanks for adding the comment (see above).